### ISSUE TITLE
Main page without links on articles texts

### ISSUE THREAD

**Issue by Ipanema1** [GavickPro/Portfolio-Free-Joomla-Template#24]
Hi, how I make same page https://demo.gavick.com/joomla3/portfolio/ with 8 pictures and 8 sliding texts (for example), only set inactive text for entry click to article. This is in code: a class="item__summary" and How can I remove "a" from your template in this 8 texts (H2 texts and also classic text from article)?  Is It Possible keep it this texts inactive? Thanks.

**Comment by dziudek** [comment]
Hello,

As you can see here - the links for the titles can be disabled via component settings: https://github.com/GavickPro/Portfolio-Free-Joomla-Template/blob/master/html/com_content/category/blog_item.php#L35

But for the content - you have to remove the first and last line from the highlighted code fragment: https://github.com/GavickPro/Portfolio-Free-Joomla-Template/blob/master/html/com_content/category/blog_item.php#L47-L55

**Comment by Ipanema1** [comment]
Yeah. It work perfect in articles! Thanks, Dziudek. Its a same in K2 items? I see different php sources for K2 items/articles...

**Comment by dziudek** [comment]
We are not supporting K2 in this template - there is no overrides for this component in our template.
